src/patternfly/sass-utilities/mixins.scss

```diff
@@ -192,3 +192,65 @@
     #{$value}: #{$prop};
   }
 }
+
+@mixin pf-label-list-builder($namespace, $row-gap: "sm", $column-gap: "sm", $grid-gap: "md") {
```

Comment:
Can you pull this out of this PR and put it in a separate issue or whatever you think is appropriate since we're just exploring the idea currently?